Make the generated script end with "exit 0" so that the "exit 1" that
our configure wrapper puts afterward gets ignored.
--- setup/WRBGCFLS.i.orig	2018-07-14 09:19:58.000000000 -0500
+++ setup/WRBGCFLS.i	2018-07-17 14:30:13.000000000 -0500
@@ -485,4 +485,8 @@
 	WriteADstFile1("my_project_d",
 		"Makefile", "", "Make file",
 		WriteBashGccMakeFile);
+
+	WriteEolToOutput();
+	WriteCStrToOutput("exit 0");
+	WriteEolToOutput();
 }
